apidef/api_definitions.go

```diff
@@ -868,6 +875,17 @@ func (a *APIDefinition) DecodeFromDB() {
 
 	makeCompatible("authToken")
 	makeCompatible("jwt")
+	// JWTScopeToPolicyMapping and JWTScopeClaimName are deprecated and following code ensures backward compatibility
+	a.JWTScopeClaim = ScopeClaim{
```

Comment:
I think you should do this only if `a.JWTScopeClaimName` is not empty but `ScopeClaimName` is empty.